packages/web/src/components/user-generated-text/UserGeneratedTextV2.tsx

```diff
@@ -313,7 +313,15 @@ export const UserGeneratedTextV2 = forwardRef(function (
 
   return (
     <Text
-      style={{ whiteSpace: 'pre-wrap' }}
+      style={{
+        whiteSpace: 'pre-wrap'
+      }}
+      css={{
+        '&::selection': {
+          backgroundColor: '#a116b7',
```

Comment:
no constant for this?

Reply:
we don't - this is actually an "out of band" color, but it's one we use elsewhere. do you want me to add to harmony?

Reply:
let me merge this and ill start a harmony thread

Reply:
yea on board with that